peerstore_test.go

```diff
@@ -25,18 +28,24 @@ func getAddrs(t *testing.T, n int) []ma.Multiaddr {
 	return addrs
 }
 
+func randPeerID() peer.ID {
```

Comment:
Not that this doesnt work or anything, but you can definitely just use random strings as peer IDs:

```
peerA := peer.ID("foo")
```